widgetastic_manageiq.py

```diff
@@ -1108,3 +1109,27 @@ def select(self, title):
     @property
     def selected(self):
         return next(btn.title for btn in self._view_buttons if btn.active)
+
+
+class Search(View):
```

Comment:
I also planned to create Search but it's quite sophisticated due to Advanced Search View.
:+1: I'm glad to see that you've implemented basic search widget.
 I hope you will take care of advanced part as well later.